### PR TITLE
[Arctic-1120]we return bytes array when we response file load request

### PR DESCRIPTION
**Pull request by Aireed**
## Brief change log
fix: https://github.com/NetEase/arctic/issues/1120

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Review by wangtaohz — APPROVED** [review]
LGTM

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/PlatformFileInfoService.java [hunk 1]
```diff
@@ -55,4 +55,12 @@ public String getFileContentById(Integer fileId) {
     }
   }
 
+  public byte[] getFileContentBytesById(Integer fileId) {
```
Comment (zhoujinsong) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/PlatformFileInfoService.java:58]: Is the method `getFileContentById ` still needed?

Comment (Aireed) [reply ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/PlatformFileInfoService.java:58]: There is no call of the method getFileContentById left. 
 i will remove it.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**